lib/trufi_models.dart

```diff
@@ -443,20 +442,44 @@ class PlanItineraryLeg {
   }
 
   String _carTypeString(TrufiLocalization localization) {
```

Comment:
I think we should rename this method, it seems that is more the transportation type.